dist/index.html

```diff
@@ -20,45 +20,21 @@ <h1 id="splashMessage">Welcome Back User!</h1>
     </div>
   </header>
   <main>
-    <section class="trip-column present-column">
+    <section class="trip-column present-column"id ="presentColumn">
```

Comment:
I like this idea of a columns layout, but how many of these trips are gonna end up being in a "present" column?